nixos/modules/services/video/mirakurun.nix

```diff
@@ -0,0 +1,155 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.mirakurun;
+  mirakurun = pkgs.mirakurun;
+  username = config.users.users.mirakurun.name;
+  groupname = config.users.users.mirakurun.group;
+  settingsFmt = pkgs.formats.yaml {};
+in
+  {
+    options = {
+      services.mirakurun = {
+        enable = mkEnableOption mirakurun.meta.description;
+
+        logLevel = mkOption {
+          type = types.ints.between (-1) 3;
+          default = 2;
+          description = "Mirakurun log level.";
+        };
+
+        domainSocket = mkOption {
+          type = with types; nullOr path;
+          default = "/var/run/mirakurun/mirakurun.sock";
+          description = "Path to domain socket used to communicate with Mirakurun.";
```

Comment:
Same for this option, do people actually need to change this ever? I'd think the default value is good enough for pretty much everybody, and we can set this with `serverSettings` below without an extra option.

Reply:
Fixed.